scripts/decredStakeVoting.md

```diff
@@ -3,45 +3,44 @@
+A ticket in the pool has a 99.5% chance of being drawn within 5 months. On average, a ticket will only wait in the pool for a month. For the fraction of a percent of tickets that expire, the corresponding staked Decred are unlocked and returned to the user who can then decide whether to spend them on another ticket or something else.
 
-Tickets that are drawn from the pool are chosen randomly. Using random entropy from the blockchain, 5 tickets are drawn.
+Tickets are drawn from the pool randomly, using random entropy generated by the blockchain. With each new block, 5 tickets are drawn to vote.
 
-When a ticket is drawn, it's owner's vote will be cast by their configured stake server.
+When a vote is drawn, it will be cast by its configured stake server which is standing by 24-7 to make sure stake voters' votes are cast.
 
-Stake servers allow the casting of votes without worry of losses due to computer downtime or bad Internet connections. In return, stake servers receive a small fee taken from vote proceeds. This server fee helps pay for the cost of server infrastructure needed to ensure stake votes are cast quickly and reliably.
+Stake servers allow the casting of votes without worry of losses due to computer downtime or bad Internet connections. In return, stake servers receive a small fee. This server fee helps pay for the cost of server infrastructure needed to ensure stake votes are cast quickly and reliably.
 
-The 5 stake votes contained in each new block vote on the validity of the previous one. If this block is deemed invalid by at least a 3 to 5 majority, then its associated transactions and proof-of-work reward are stripped from the network while the proof-of-stake reward is retained. This process acts as a check on miner power, giving stakeholders a way to crack down on miners who are not following the rules and prevent miner aggressions such as block withholding or a 51% attack.
+In rare cases, it is possible for votes to be missed even though they were cast. This can happen when miners are processing blocks faster than the votes can propagate through the network. This leads to the mining of blocks with less than 5 votes. If a block is mined with less than the required 5 votes, the remaining votes destined for that block are considered ‘missed’ and the funds used to purchase them are unlocked.
 
-A ticket may also cast votes concerning other agenda items put up for a community vote. One example of an item that was approved in this manner was a consensus rule upgrade improving the algorithm which sets ticket price.
+After a stake voter's ticket has been drawn and their votes are successfully cast, the Decred used for the ticket is unlocked and a reward for voting, called a ticket reward, is deposited into the ticket owners account. This ticket reward amounts to 6% of the total Decred block reward. After a waiting period of 20 hours, these funds become available for spending.
```

Comment:
Lets do the start of this paragraph first, then the above paragraph about the possible edge case, then last we get into more details about the rewards.